Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py

```diff
@@ -70,10 +74,7 @@ def trigger_generic_webhook(options):
     print(f"Detected Pr: {pr_title=}, {pr_link=}, {pr_body=}")
 
     # Handle cases where the PR did not intend to add links:
-    if ("fixes:" not in pr_body.lower()
-            and "relates:" not in pr_body.lower()
-            and "fixed:" not in pr_body.lower()
-            and "related:" not in pr_body.lower()):
+    if all(x.casefold() not in pr_body.casefold() for x in ("fixes:", "relates:", "fixed:", "related:")):
```

Comment:
`fixed:` and `related:` are not in the regex, so are these cases even valid? (need to check)

Reply:
It is important not to break backward, and until now, all 4 options have been viable.